tests/functional/registry/test_config.py

```diff
@@ -29,15 +29,29 @@ def test_registry_setGovernance(gov, registry, rando):
         registry.acceptGovernance({"from": gov})
 
 
-def test_banksy(gov, registry, create_vault, rando):
+def test_banksy(gov, guardian, rewards, registry, create_token, create_vault, rando):
+
+    # Not just anyone can create a new Vault, only a Banksy can!
+    with brownie.reverts():
+        registry.newVault(create_token(), guardian, rewards, "", "", {"from": rando})
```

Comment:
This test is in the wrong place.
```suggestion
    # Not just anyone can create a new endorsed Vault, only governance can!
    with brownie.reverts():
        registry.newVault(create_token(), guardian, rewards, "", "", {"from": rando})
```